tests/perf-system/submitter/handle_arguments.h

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+
+#pragma once
+
+#include "ccf/ds/logger.h"
+
+#include <CLI11/CLI11.hpp>
+#include <iostream>
+
+class ArgumentParser
+{
+public:
+  std::string label; //< Default set in constructor
+  std::string pid_file; //< Default set in constructor
+
+  std::string cert;
+  std::string key;
+  std::string rootCa;
+  std::string server_address = "127.0.0.1:8000";
+  std::string send_filepath = "../tests/perf-system/submitter/cpp_send.parquet";

```

Comment:
The following three default values are... very specific? I'm not sure I like the idea of having default output paths in particular, it seems like it makes it a bit too easy to accidentally overwrite some results?

Reply:
You are talking about the three filepaths, correct? I think you are right, I will remove them.